state/indexer/tx/kv/kv_test.go

```diff
@@ -195,7 +195,7 @@ func TestTxSearchDeprecatedIndexing(t *testing.T) {
 
 	err = b.Set(depKey, hash2)
 	require.NoError(t, err)
-	err = b.Set(keyFromHeight(txResult2), hash2)
+	err = b.Set(KeyFromHeight(txResult2), hash2)
```

Comment:
why did we make this function public?

Reply:
Because `tx/kv` and `sink/kv`  both use this function in the unit test.